Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,165 @@
+# Hide shellcheck complaint:
+# shellcheck source=/dev/null
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+VENDOR_DIR="$HOMEBREW_LIBRARY/Homebrew/vendor"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  if [[ "$HOMEBREW_PROCESSOR" = "Intel" ]]
+  then
+    # TODO test only. Upload files to bintray before shipping !!!
+    ruby_URL="https://github.com/Homebrew/homebrew-portable/releases/download/20160512/portable-ruby-2.0.0-p648.i386-darwin9.tar.gz"
+    ruby_SHA="035ccdc2c073172148c5ffe3e7004011862b400c6aa3a2831c1cec46a0876de5"
+    git_URL="https://github.com/Homebrew/homebrew-portable/releases/download/20160506/portable-git-2.8.2.universal-darwin9.tar.gz"
+    git_SHA="54ce39f5269202dbf08281fb7d0bbf73f03ef38b02a74e8f6ab84467bb70c793"
+  else
+    ruby_URL=""
+    ruby_SHA=""
+    git_URL=""
+    git_SHA=""
+  fi
+else
+  ruby_URL=""
+  ruby_SHA=""
+  git_URL=""
+  git_SHA=""
+fi
+
+fetch() {
+  local curl_args
+  local sha
+  local temporary_path
+
+  if [[ -n "$HOMEBREW_VERBOSE" ]]
+  then
+    curl_args="-fLA"
+  elif [[ -z "$HOMEBREW_QUIET" ]]
+  then
+    curl_args="-#fLA"
+  else
+    curl_args="-sfLA"
+  fi
+
+  temporary_path="${CACHED_LOCATION}.incomplete"
+
+  mkdir -p "$HOMEBREW_CACHE"
+  [[ -z "$HOMEBREW_QUIET" ]] && echo "==> Downloading $VENDOR_URL"
```

Comment:
A minor detail, but could be relevant in a context where the exit code of such a line is used. If the condition isn't met, this returns exit code 1 indicating failure. If you invert the condition and use `||` instead of `&&`, then the exit code will always indicate success. In this particular example:

```sh
[[ -n "$HOMEBREW_QUIET" ]] || echo "==> Downloading $VENDOR_URL"
```